src/_guides/language/index.md

```diff
@@ -31,6 +31,10 @@ These two resources are popular with both beginning Dartisans and experts.
 : A list of common questions for those who are interested in stronger
   static typing.
 
```

Comment:
This list is getting long. I'd break it up into groups. Maybe bulleted lists?

* [Sound Dart](...): How and why...
  * [FAQ](...): A list...
  * [Common Problems](...): ...
* Customize Static Analysis: ...
* Asynchronous programming
  * Futures
  * Streams
* Language articles
* Specification